mappings/net/minecraft/block/pattern/BlockPatternBuilder.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS none/asr net/minecraft/block/pattern/BlockPatternBuilder
+	METHOD a start ()Lnone/asr;
+	METHOD a where (CLcom/google/common/base/Predicate;)Lnone/asr;
+	METHOD a aisle ([Ljava/lang/String;)Lnone/asr;
```

Comment:
Should probably have a better name, taken from https://twitter.com/Dinnerbone/status/477358451115720704